definitions/features/downstream.rb

```diff
@@ -30,6 +30,17 @@ def setup_repositories(version)
     end
   end
 
+  def absent_repos(version)
+    execute!(%(subscription-manager refresh))
+    all_repos = execute!(
+      'subscription-manager repos --list | ' \
```

Comment:
Just of safety reasons, we should ensure we are running in english locale `LANG=en_US.utf-8`

Reply:
Sure @iNecas 
Is the following change relevant to what you meant above?
Or setting english locale `LANG=en_US.utf-8` is actually something different.
```
-      'subscription-manager repos --list | ' \
-        "awk -F':' '/Repo ID/{gsub(/ /, \"\", $2); print $2}'"
+      %('subscription-manager repos --list | ') \
+        %("awk -F':' '/Repo ID/{gsub(/ /, \"\", $2); print $2}'")
```

Reply:
The reason why I'm suggesting `LANG=en_US.utf-8` is, that in case folks have different localization on the sytem, the subscription manager can include something else, than `Repo ID` in the output, in which case it could fail. I'm not sure it would happen, but it's better to be on safe side.

See what we do with hammer `https://github.com/theforeman/foreman_maintain/blob/531e504360858b0df97bf49c24087bb4d43c94d9/lib/foreman_maintain/utils/hammer.rb#L70`